assets/js/util/date-range.js

```diff
@@ -30,13 +30,17 @@ import { STORE_NAME as CORE_USER } from '../googlesitekit/datastore/user/constan
 /**
  * Gets the current dateRange string.
  *
- * @param {string} [dateRange] Optional. The date range slug.
+ * @param {string} [dateRange]      Optional. The date range slug.
+ * @param {boolean} [returnNumber]  Optional. If true, returns the number only.
  * @return {string} the date range string.
  */
-export function getCurrentDateRange( dateRange = getCurrentDateRangeSlug() ) {
+export function getCurrentDateRange( dateRange = getCurrentDateRangeSlug(), returnNumber = false ) {
```

Comment:
Having boolean parameters to change function behavior is not a good practice because it violates the principle of functions being responsible for a single task, so the need for this indicates we need to split something out of here instead. We can introduce a function like `getCurrentDateRangeDayCount` or similar, which does this.